components/Layout/NavMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, useState } from "react";
+import {
+  Box,
+  useColorModeValue,
+  NAMED_COLORS,
+  Portal,
+  useConst,
+} from "@ironfish/ui-kit";
+
+import NavListOfLinks from "./NavListOfLinks";
+
+interface NavMenuProps {
+  menuRef: RefObject<FC | null>;
+  children: ReactNode;
+}
+
+const NavMenu: FC<NavMenuProps> = ({ menuRef, children }) => {
+  const colors = useColorModeValue(
+    { bg: NAMED_COLORS.DEEP_BLUE },
+    { bg: NAMED_COLORS.WHITE }
+  );
+  const burgerLineStyle = useConst({
+    width: "2.0625rem",
+    height: "0.1719rem",
+    transition: "all .3s ease-in-out",
+  });
```

Comment:
What is different between such using:
```tsx
const burgerLineStyle = {
    width: "2.0625rem",
    height: "0.1719rem",
    transition: "all .3s ease-in-out",
  }
```
Also it can be used as Global variable